experimental/addons/redis-exporter/README.md

```diff
@@ -0,0 +1,191 @@
+      properties:
+        repoType: "helm"
+        url: "https://charts.bitnami.com/bitnami"
+        chart: "redis"
+        version: "17.7.3"
+        values:
+          global:
+            redis:
+              password: <password>
+          master:
+            persistence:
+              size: 16Gi
+          replica:
+            persistence:
+              size: 16Gi
+
+```
+
+#### 2. Create Redis Secret
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <redis-secret-name>
+data:
+  username: <username-bash64> # default: ""
+  password: <password-bash64> # default: password in base64
+```
+
+#### 3. Create Redis Exporter Server
+
+```yaml
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: redis
+  namespace: default
+spec:
+  components:
+    - type: redis-exporter-server
+      name: redis-exporter
+      properties:
+        address: <host>:<port>
+        secretName: <redis-secret-name>
+        disableAnnotation: false
+        name: redis-server-exporter
+  traits:
+    - type: prometheus-scrape
+      properties:
+        port: <port>
+        path: /metrics
+  workflow:
+    steps:
+      - type: import-grafana-dashboard
+        name: import-redis-exporter-dashboard
+        properties:
+          uid: redis-exporter-dashboard
+          title: Redis Exporter Dashboard
+          url: https://github.com/oliver006/redis_exporter/blob/master/contrib/grafana_prometheus_redis_dashboard.json
+```
+
+### Work as a trait.
+
+#### 1. Create Redis Secret
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <redis-secret-name>
+data:
+  username: <username-bash64> # default: ""
+  password: <password-bash64> # default: password in base64
+```
+
+#### 2. Create Redis Instance
+
+```yaml
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: redis
+  namespace: default
+spec:
+  components:
+    - type: helm
+      name: redis
+      properties:
+        repoType: "helm"
+        url: "https://charts.bitnami.com/bitnami"
+        chart: "redis"
+        version: "17.7.3"
+        values:
+          global:
+            redis:
+              password: <password>
+          master:
+            persistence:
+              size: 16Gi
+          replica:
+            persistence:
+              size: 16Gi
+      traits:
+        - type: redis-exporter
+          properties:
+              address: <host>:<port>
+              secretName: <redis-secret-name>
+              disableAnnotation: false
+              name: redis-server-exporter
+        - type: prometheus-scrape
+          properties:
+              port: <port>
+              path: /metrics
+  workflow:
+    steps:
+      - type: import-grafana-dashboard
+        name: import-redis-exporter-dashboard
+        properties:
+          uid: redis-exporter-dashboard
+          title: Redis Exporter Dashboard
+          url: https://github.com/oliver006/redis_exporter/blob/master/contrib/grafana_prometheus_redis_dashboard.json
```

Comment:
you need to add a deploy step to make the component deployed when using workflow

Reply:
Sorry, it is my fault, I will fix it.